newsrevenuehub/rev-engine
# DEV-1416 Run "npm audit fix" on rev-engine ./spa

Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)
To upgrade packages and reduce vulnerabilities in spa
https://news-revenue-hub.atlassian.net/browse/DEV-1416

#### What's this PR do?
fixes spa package issues

#### Why are we doing this? How does it help us?
fix security issue/ use latest more optimal packages for react app

#### How should this be manually tested?
Check for any things that break manually by browsing the front end app.

The branch should also be locally built and run to see all is ok.

#### Have automated unit tests been added?
"N/A"

#### How should this change be communicated to end users?
"N/A"

#### Are there any smells or added technical debt to note?
Yes, semantic.min.css is inserted in index.html. Otherwise, build breaks due to incompatible packages

#### Has this been documented? If so, where?
Details are in JIRA ticket DEV-1416 


#### What are the relevant tickets?
[DEV-1416]

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?

#### Are there next steps? If so, what? Have tickets been opened for them?


[DEV-1416]: https://news-revenue-hub.atlassian.net/browse/DEV-1416?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
